data/plugins/findfile.lua

```diff
@@ -19,10 +19,15 @@ command.add(nil, {
       local start, total = system.get_time(), 0
       for i, project in ipairs(core.projects) do
         for project, item in project:files() do
-          if complete or #files > config.plugins.findfile.file_limit then return end
+          if #files > config.plugins.findfile.file_limit then 
+            core.command_view:update_suggestions() 
+            return 
+          end
+          if complete then return end
```

Comment:
Might be a good idea to test `complete` first.

Reply:
UGH.